router/router.go

```diff
@@ -219,12 +220,16 @@ func New(cfg *config.Configuration) (r *Router, err error) {
 	exchanges = newExchangeMap(cfg)
 	theExchange := exchange.NewExchange(theClient, pbc.NewClient(&cfg.CacheURL), cfg, r.MetricsEngine, bidderInfos, gdprPerms)
 
-	openrtbEndpoint, err := openrtb2.NewEndpoint(theExchange, paramsValidator, fetcher, cfg, r.MetricsEngine, pbsAnalytics, disabledBidders, defReqJSON, bidderMap)
+	currencyConverter := currencies.NewRateConverter(theClient, cfg.CurrencyConverter.FetchURL, time.Duration(cfg.CurrencyConverter.FetchIntervalSeconds)*time.Second)
+
+	openrtbEndpoint, err := openrtb2.NewEndpoint(theExchange, paramsValidator, fetcher, cfg, r.MetricsEngine, pbsAnalytics, currencyConverter, disabledBidders, defReqJSON, bidderMap)
```

Comment:
The code is quite a bit simpler if you send the `currencyConverter` directly into `NewExchange`, rather than through `NewEndpoint` and `NewAmpEndpoint`.

Do any of your future changes planning to use the currency conversions in the endpoint code? I can't think of any reason you'd want to do that... but let me know if I'm missing something.